app/models/group.rb

```diff
@@ -0,0 +1,4 @@
+class Group < ActiveRecord::Base
+	belongs_to :company
+	has_many :group_members
```

Comment:
what is group_member? another module? why need this?

Reply:
Needed to store which member is in which group inside database. contains group_id, user_id pairs.